components/project/post/InterestsPage.tsx

```diff
@@ -40,7 +40,7 @@ export default ({ project }): JSX.Element => {
           <span style={{ fontWeight: 'bold' }}>{`Areas of Interest`}</span>
           <span style={{ color: COLORS.GRAY_BB }}>{` (max 5)`}</span>
         </Typography>
-        <ArrayForm label="Interests" items={interests} setItems={setInterests} limit={5} />
+        <ArrayForm label="Interests" items={interests} setItems={setInterests} limit={5} options={interests} />
```

Comment:
variable "interests" isn't the options from the backend, it's the state of the project's interests.
- need to add options prop in line 26, then pass that into ArrayForm's options
- modify pages/project/post to fetch from backend (I did teamSize and duration choices for example)